Source/Controllers/SubviewControllers/SPTablesList.m

```diff
@@ -741,6 +741,24 @@ - (IBAction)togglePinTable:(nullable id)sender {
     // actual pin toggle will happen when notification is received and processed
 }
 
+
+- (IBAction)toggleCopyTableName:(nullable id)sender {
```

Comment:
I would use either just `copyTableName` (which might be missleading) or `storeTableNameToPasteboard` (recommended)

Reply:
I have updated it with `copyTableName `.
If you accept this PR, you can continue to change it with any change you want.
I'm not very good at ObjectC development, I'm just providing some ideas and hoping that it will be better.